audonnx/core/model.py

```diff
@@ -223,17 +224,31 @@ def __call__(
         the output of that node is returned.
         Otherwise a dictionary with outputs of all nodes is returned.
 
+        If ``concat`` is set to ``True``,
+        the output of the requested nodes is concatenated
+        and a single array is returned.
```

Comment:
Maybe mention that it is concatenated along the last non-dynamic axis or is this obvious anyway?

Reply:
Makes sense, changed to:

![image](https://user-images.githubusercontent.com/10383417/175009530-8b2352eb-3a07-4166-8b2f-8cc5b4c9d2d1.png)